app/bundles/FormBundle/EventListener/FormSubscriber.php

```diff
@@ -197,6 +197,15 @@ public function onFormSubmitActionSendEmail(Events\SubmissionEvent $event)
             $this->mailer->setSubject($config['subject']);
 
             $this->mailer->addTokens($tokens);
+
+            if (isset($config['templates'])) {
+                /** @var \Mautic\EmailBundle\Model\EmailModel $model */
+                $model = $this->mailer->getFactory()->getModel('email');
```

Comment:
Please inject the email model directly into the FormSubscriber class using a constructor parameter. It also needs to be added to the FormSubscriber config in the bundles/FormBundle/Config/config.php section that defines the parameters for the FormSubscriber (under the `events` key).